1-js/12-generators-iterators/2-async-iterators-generators/article.md

```diff
@@ -1,48 +1,49 @@
 
-# Async iteration and generators
+# Iteração assíncrona e geradores
 
-Asynchronous iteration allow us to iterate over data that comes asynchronously, on-demand. Like, for instance, when we download something chunk-by-chunk over a network. And asynchronous generators make it even more convenient.
+A iteração assíncrona permite que iteremos sobre dados que são gerados de forma assíncrona, sob demanda. Por exemplo, ao baixarmos algo pedaço por pedaço pela rede. E os geradores assíncronos tornam isso ainda mais conveniente.
 
-Let's see a simple example first, to grasp the syntax, and then review a real-life use case.
+Vamos ver primeiro um exemplo simples para compreender a sintaxe e, em seguida, revisaremos um caso de uso da vida real.
 
-## Recall iterables
+## Relembrando iteráveis
```

Comment:
It must not use the gerund because the original text does not refer to this, so we should translate as **Relembrar os iteráveis**.